experimental/dds/tree2/src/core/rebase/index.ts

```diff
@@ -32,3 +34,6 @@ export {
 	Violation,
 } from "./verifyChangeRebaser";
 export { findAncestor, findCommonAncestor, rebaseBranch, rebaseChange } from "./utils";
+// TODO: This is moved here temporarily to avoid a circular dependency. The RepairDataStore will be removed completely soon.
```

Comment:
@yann-achard-MS This is the reasoning for the move. Let me know if you want me to do something else though.

Reply:
What is the circular chain in this case?

Reply:
Repair needs rebase now because that's where we're exposing change atom id (also changed this in the README chart). I figured this would be reasonable since we'll be removing the repair stuff that rebase needs.

Reply:
I just wasn't sure whether there were more links in the chain that forms the cycles, which made it hard to evaluate things like "Should some of these link be merged into a single directory?". After looking it up I can tell that's not the case. I agree this is fine. Marking this as resolved.